src/test/java/uk/gov/verifiablelog/merkletree/MerkleTreeTests.java

```diff
@@ -272,7 +270,7 @@ public void should_useStoreToRetrieveAndSave_for_emptyTreeAndEmptyStore() {
     }
 
     @Test
-    public void should_useStoreToRetrieveAndSave_for_treeWithLeaves() {
+    public void currentRoot_should_useStoreToRetrieveAndSave_for_treeWithLeaves() {
```

Comment:
could be `currentRoot_usesStoreToRetrieveAndSave_for_treeWithLeaves` which is a little shorter